### PR TITLE
What is git

### PR DESCRIPTION
**Pull request by palmcm**

### CODE REVIEW

**Review by saenyakorn — COMMENTED** [review]
🤔

**Review by bombnp — COMMENTED** [review]
s u s p i c i o u s

File: Git.md [hunk 1]
```diff
@@ -0,0 +1 @@
+GIT = Genshin ImpacT
```
Comment (suphon-t) [inline Git.md:1]: 👎👎👎

**Review by bombnp — APPROVED** [review]

**Review by suphon-t — CHANGES_REQUESTED** [review]
Not the correct definition of Git

**Pull request closed without merging**